sympy/matrices/tests/test_matrices.py

```diff
@@ -1897,7 +1897,7 @@ def test_errors():
     raises(IndexError, lambda: eye(3)[5, 2])
     raises(IndexError, lambda: eye(3)[2, 5])
     M = Matrix(((1, 2, 3, 4), (5, 6, 7, 8), (9, 10, 11, 12), (13, 14, 15, 16)))
-    raises(ValueError, lambda: M.det('method=LU_decomposition()'))
+    raises(ValueError, lambda: M.det(method='LU_decomposition()'))
```

Comment:
This is not the test case we're looking for. We're looking for a test case for `det` function, not the `.det` method.

Reply:
This is a corrected test case. Since .det uses a `method = "....."` argument, that example popped up an error during local tests. So, I changed into `method=method` format.

Reply:
Okay. Thanks.